### ISSUE TITLE
Lime 6 -minify breaks robotlegs

### ISSUE THREAD

**Issue by TomByrne** [robotlegs/openfl-robotlegs-framework#21]
Using the -minify option when building with Lime 6.x seems to break robotlegs.

This is likely caused by dead code elimination, or something else which is removing information that robotlegs needs.

The workaround for the moment is to just not use -minify, but I will look at a better solution soon.

**Comment by peteshand** [comment]
Sounds about right.. there is a lot of Reflection being used. Do you get the same result when turning DCE to full?

**Comment by TomByrne** [comment]
Yeah, but that doesn't say too much as essentially none of the 'configure' classes get called after the robotlegs context is set up.

**Comment by peteshand** [comment]
So if they are behaving the same, I'd suggest setting DCE to full (with no minify) and then in the output JS check if the classes past to .configure have made it through the compiler and if so what functions are present. If some or all of the class is missing try adding the @keep mata tag. Not really a long term solution, but will give a little extra info
